src/config.js

```diff
@@ -253,3 +247,19 @@ export const DRAWING_HIT_TOLERANCE = 6
 export const VECTOR_TILES_STYLE_ID = 'ch.swisstopo.leichte-basiskarte_world.vt'
 
 export const VECTOR_TILES_STYLE_URL = `https://vectortiles.geo.admin.ch/styles/${VECTOR_TILES_STYLE_ID}/style.json`
+
+/**
+ * Display a big developpment banner on all but these hosts.
+ *
+ * @type {String[]}
+ */
+ export const NO_WARNING_BANNER_HOSTNAMES = ['map.geo.admin.ch']
```

Comment:
On test.map.geo.admin.ch the other dev site warning should be removed:
![image](https://user-images.githubusercontent.com/67745584/208600608-3a44e4ab-af24-4651-bd22-b7cc1111b6f9.png), you simply need to add `test.map.geo.admin.ch` to this array.

Reply:
OK, corrected.